tests/numpy_test.py

```diff
@@ -118,3 +118,12 @@ def test_concatenate() -> None:
     d3: np.ndarray[np.int32] = np.concatenate((d, d), axis=0)
     scalar: np.float32 = np.float32(3.0)
     assert isinstance(scalar, np.float32)
+
+
+def test_at_least_2d() -> None:
+    arr: np.ndarray[np.float64] = np.atleast_2d(3.0)
+    assert isinstance(arr, np.ndarray)
+    assert_dtype(arr, np.float64)
+
+    a = np.atleast_2d(1, [1, 2], [[1, 2]], 1)
```

Comment:
Very minor improvement. Not worth delaying merging for.

```suggestion
    a: List[np.ndarray[np.int64]] = np.atleast_2d(1, [1, 2], [[1, 2]], 1)
```